components/_modals.scss

```diff
@@ -0,0 +1,118 @@
+/*
+
+Stylesheet for all things related to presentation of 
+modals across the website.
+
+*/
+
+// Overlay behind the modal
+
+.ui-widget-overlay {
+	background: $tws-yellow;
+}
+
+// General modal box positioning
+
+.ui-dialog {
+
+	position: fixed !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered left, overflow, padding, position, top, -moz-transform, -ms-transform, -o-transform, -webkit-transform, transform